site/gatsby-site/src/components/checklists/ChecklistsIndex.js

```diff
@@ -47,43 +122,56 @@ const ChecklistsIndex = ({ users }) => {
           <h1 className="mr-auto">
             <Trans>Risk Checklists</Trans>
           </h1>
-          {loggedIn && (
-            <Button
-              id="new-checklist-button"
-              onClick={async () => {
-                const newChecklist = {
-                  id: generateId(),
-                  owner_id: user.id,
-                  name: 'Unspecified System',
-                  about: '',
-                  tags_goals: [],
-                  tags_methods: [],
-                  tags_other: [],
-                  risks: [],
-                };
-
-                try {
-                  await insertChecklist({
-                    variables: { checklist: newChecklist },
-                  });
-                } catch (error) {
-                  addToast({
-                    message: t('Could not create checklist.'),
-                    severity: SEVERITY.danger,
-                    error,
-                  });
-                  return;
-                }
-                window.location = '/apps/checklists?id=' + newChecklist.id;
-              }}
-            >
-              <Trans>New</Trans>
-            </Button>
-          )}
+          <div className="flex gap-2 items-center">
```

Comment:
We should use the `<ListSkeleton />` component instead of `<Spinner />` component in the loading state. We should also improve the empty list layout, maybe with an indicating message of the empty state.

<img width="953" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/090bcac9-cf4a-44dc-a892-2d44ebd22296">

Reply:
Skeletons:

![image](https://github.com/responsible-ai-collaborative/aiid/assets/25443411/34f5b2a0-7265-44a1-81e8-c482579eefc1)


Empty list:

![image](https://github.com/responsible-ai-collaborative/aiid/assets/25443411/0e459f81-a111-4f80-b1e0-28ee0a661992)